source/guides/contributing/layout.md

```diff
@@ -0,0 +1,15 @@
+Denizen's Project Layout
+------------------------
+
+```eval_rst
+.. contents:: Table of Contents
+    :local:
+```
+
+**DenizenScript** is a language run by **Denizen**. The most popular implementation of Denizen is the Spigot implementation, which is the jar you've most likely downloaded many times. As of the time of writing, it is the *only* maintained impl.
```

Comment:
I have no idea what the sentence `**DenizenScript** is a language run by **Denizen**` means.

Reply:
Clarified following a Discord conversation